.gitignore

```diff
@@ -1,6 +1,8 @@
 build/
 .vscode/
 
+remake.sh
```

Comment:
Since we are removing this lines on a later commit, let's not add them here.

Reply:
Fixed!